.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.13"]
+        python-version: ["3.10", "3.13"]
         runs-on: [ubuntu-latest, windows-latest, macos-14]
 
         include:
```

Comment:
You can also remove this if you want to:

```
        include:
          - python-version: "pypy-3.10"
            runs-on: ubuntu-latest
```

I don't have any experience with PyPy and wouldn't know how to debug those PyPy specific issues. But if you know what's up with it and know how to get it to pass, also feel free to keep it